core/src/main/java/com/nextdoor/bender/InternalEvent.java

```diff
@@ -46,6 +49,25 @@ public InternalEvent(String eventString, Context context, long arrivalTime) {
     this.eventSha1Hash = DigestUtils.sha1Hex(this.eventString);
     this.arrivalTime = arrivalTime;
     this.eventTime = arrivalTime;
+
+    this.metadata.put("eventEpochMs", new Long(this.getEventTime()));
```

Comment:
Doesn't this happen inside of `getEventTimeI()` at https://github.com/Nextdoor/bender/pull/53/files#diff-854193347bb2bd99f4de19814c41a184R150?